sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="view_sales_order_line_input_tree" model="ir.ui.view">
```

Comment:
Indent properly this (1 indentation level), so that IDEs on new lines behave well.